uwblueprint/childrens-aid-society
# Create Individual Details Entrypoint

## Notion ticket link
[Build out Individual Details flow entrypoint](https://www.notion.so/uwblueprintexecs/Build-out-Individual-Details-flow-entrypoint-8c88ac1156fe4e74984111ecd9671295)


## Implementation description
* Created new IndividualDetailsEntry and PromptBox components
* React router link to Individual details entry page



## Steps to test
1. Go to http://localhost:3000/intake
2. Press next button until individual details entry is reached
3. Verify design aligns with [Figma](https://www.figma.com/file/1obAgfYZ639hpKXV1ZP7hJ/3.-FeatureSets-%3E-Case-intake?node-id=1175%3A19815)
4. Verify add child button navigates to new page
5. Verify add caregiver button navigates to new page



## What should reviewers focus on?
* Naming for PromptBox component
* Where should UserPlusIcon be in the repo


## Checklist
